### ISSUE TITLE
Uninstall Network Manager from the VM

### ISSUE THREAD

**Issue by nikhilh** [onstutorial/onstutorial#4]
The Network Manager automatically starts a DHCP client for each interface (including the switch interfaces). This leads to spurious packet-ins with DHCP request packets. The Network Manager should be uninstalled to keep things clean:
sudo apt-get remove network-manager

We however need DHCP to run for the two primary interfaces. So, add the following lines to /etc/network/interfaces:
auto eth0
iface eth0 inet dhcp

auto eth1
iface eth1 inet dhcp

**Comment by daviderickson** [comment]
Done, mine only has eth0 however, does yours have more than one 
interface?  I can't think of a reason it would need to.

**Comment by nikhilh** [comment]
Yes, I'm using the config of the old VM: (1) a NAT'ed eth0 to talk to the
outside world, and (2) a host-only eth1 to SSH into.

-/\/

**Comment by brandonheller** [comment]
+1, I removed the text on setting up the host-only port from the
instructions, because that's no longer needed.

If the interface doesn't exist, I don't think will harm anything though.
 Might as well leave it in.

**Comment by daviderickson** [comment]
So we've completely erased everything about SSH into the VM for this 
tutorial.  There is a single NAT'd NIC, and they have actual terminal 
windows in front of them inside the window manager of the VM.  Are you 
using the VM I posted?
